webapp/src/routes/home/instruction/components/icon/icon.component.js

```diff
@@ -0,0 +1,24 @@
+import React, { PureComponent } from 'react';
```

Comment:
nie widzę specjalnie zastosowania dla tego komponentu, skoro on tylko renderuje jeden `styled-component` no to można po prostu ten komponent utworzyć w ramach `instruction.styles`